politeiawww/pi/pi.go

```diff
@@ -50,6 +50,10 @@ func New(cfg *config.Config, pdc *pdclient.Client, udb user.Database, s *session
 		nameLengthMin      uint32
 		nameLengthMax      uint32
 		nameSupportedChars []string
+		amountMin          uint64
+		amountMax          uint64
+		endDateMax         uint64
```

Comment:
You changed this to a `int64` in politeiad but have it as an `uint64` here.  These need to be consistent.